kotlin-jvm/src/main/kotlin/playground/FuelHttp.kt

```diff
@@ -0,0 +1,19 @@
+package playground
+
+import com.github.kittinunf.fuel.core.FuelManager
+import java.net.URL
+
```

Comment:
Please add a kdoc witth the URLs present in the issue https://github.com/LouisCAD/kotlin-libraries-playground/issues/75